src/main/java/org/folio/service/pieces/flows/delete/PieceDeleteFlowManager.java

```diff
@@ -79,4 +85,26 @@ protected Future<Void> updatePoLine(PieceDeletionHolder holder, RequestContext r
       : pieceDeleteFlowPoLineService.updatePoLine(holder, requestContext);
   }
 
+  public Future<List<Void>> batchDeletePiece (PieceCollection entity, RequestContext requestContext) {
+    List <String> ids = new ArrayList<>();
```

Comment:
we should keep an existing logic that Delete by Id endpoint uses with many existing checks.
In the simplest way you can invoke this method in cycle:
Future<Void> deletePiece(String pieceId, boolean deleteHolding, RequestContext requestContext) 
It is not optimezed solution of course, but can be as starting point

Reply:
Sure, thank you for pointing it out, will do! :)